src/kvstore/src/main.rs

```diff
@@ -85,6 +110,20 @@ struct PutOpt {
     stdin: bool,
 }
 
+#[derive(Parser)]
+struct DisableOpt {
+    /// The key to disable.
+    key: String,
+
+    /// If the key is a hexadecimal string, pass this flag.
+    #[clap(long)]
+    hex_key: bool,
```

Comment:
Should be mutually exclusive with `key`.

Reply:
Same here, `hex_key` is a flag.